### ISSUE TITLE
SUSE Support pages and info should only be available to Rancher administrators

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#2895]
If the UI is in 'supported' mode, then we should only show the support button in the top-level slide in to Rancher Admins - these are users that have the 'edit settings' permission.

**Comment by nwmac** [comment]
Also - we should add a setting to allow admins to change the support URL - if set, we should use that for users, rather than remove the support button for non-admins

**Comment by aaronRancher** [comment]
Tested on Rancher `master-head` `version 618c9ef` with the following steps:
As admin, verify `Get Support` link is seen in the top-level slide.
Create a standard user.
Log into Rancher as standard user.
Standard user is able to see`Get Support` link  
![Screen Shot 2021-06-01 at 4 22 22 PM](https://user-images.githubusercontent.com/57409052/120401789-8f301080-c2f5-11eb-81f9-e466fc613e38.png)

I did not find a setting to allow admins to change the support URL.
Reopening for fix.
